### DISCUSSION TITLE
PHP Support

### DISCUSSION THREAD

**Discussion by Lauras-Universe** [zed-industries/zed#7959] [Feature Requests]
Have this native PHP Support for developing easy PHP?

**Comment by SomeoneToIgnore** [comment]
There are two good sources to learn about Zed's language support: 
* the documentation https://github.com/zed-industries/zed/tree/main/docs/src/languages
* issue search: https://github.com/zed-industries/zed/issues?q=is%3Aissue+is%3Aopen+php and https://github.com/zed-industries/zed/issues?q=is%3Aissue+is%3Aclosed+php , https://github.com/zed-industries/zed/issues/5652 specifically.

As you can see, there's some PHP support, but it has some bugs (as other languages support too).
Not sure what "developing easy PHP" is, so cannot comment that part.
